### DISCUSSION TITLE
Data curator - Get domain-specific and cross-domain context to a data entity

### DISCUSSION THREAD

**Discussion by KlausIllmayer** [pair-cg/general-contributions#53] [User Stories]
# 1. Need:
As a data curator for a performance database I like to easily gather information on the context of data entities like persons, places, works when I create a new dataset with metadata to a performance.
I like to easily enrich my new dataset and the data entities that I recognize there with context from other databases.
Thus, if I don't have information on a data entity I like to have the possiblity to run a query against databases from the performance/theatre domain and other domains (like literature domain to gather data on authors of works) that returns back recommendations about the data entity in question (kind of "named entity recognition" system).

# 2. Action:
In the backend of the database when a data curator creates a new dataset, it should be possible to use an autocomplete mechanism that looks for data from other databases.
If the data curator enters the name of a person or a stage that is not already in the database, it should be possible to use the federated search to get recommendations from other databases on the person/stage/etc. in question
It should be also possible to contact databases outside the performance/theatre domain to apply more context to a performance, e.g. a database on drama could deliver back the provided roles so that the data curator does not need to enter them once again but only check if they exist in the performance and enter the name of the actors/acresses that play that role.

# 3. Desired Outcome:
If a database already has background and contextual information on a data entity, I get the relevant information from there and don't need to enter it once again. It should also give relations to authority files (like VIAF - http://viaf.org/) and would allow to clearly indicate that data entities between different databases are the same.
The mechanism could also help in showing users more background information on a dataset (like the context information that Google gives in a dedicated box if you search for a well known data entity) without entering this data in the database.
For data curators it would help to reduce the time for searching background information when re-using already existing data.
For the performance/theatre domain specific search it could act as a NER (named entity recognition) service.

# 4. Perspective:
Data curator

# 5. Stakeholder Group:
Data providers
Data curators

# 6. Additional info:
* This surely needs a lot of programming effort at the side of the data provider to get a seamless way to use a federated search for gaining a greater context.
* It also implies, when it comes to data outside the performance/theatre domain to define mappings to common data models from this domains. It could be useful to think about using very abstract ontologies like CIDOC CRM to enable such cross-domain mapping.
* When it comes to the NER service it would be also useful to define together recommendations on how to weighten results from authority files for the performance/theatre domain (e.g. the German integrated authority file GND https://www.dnb.de/EN/Professionell/Standardisierung/GND/gnd_node.html gives information on occupations/professions of persons and subjects relevant to this person, some of this information is very specific to theatre/performance and the availability of such information could lead to a domain specific weightening for search results coming from this or other authority files)

**Comment by mariel-marshall** [comment]
Hey @KlausIllmayer - thanks for sharing! Is this something you've already built or are working on [https://theadok.at](url)? In terms of a User Story, I'm wondering if you can get into more specifics:

Can you give a concrete example that describes your process of adding a new "performance" or "person" to your database? 
What information do you need to collect about a data entity in your database and why?
